in_toto/model_test.go

```diff
@@ -283,7 +284,7 @@ func TestMetablockVerifySignature(t *testing.T) {
 	}
 	err := mb.VerifySignature(key)
 	if err != nil {
-		t.Errorf("Metablock.VerifySignature returned '%s', expected nil", err)
+		t.Errorf("Metablock.VerifyRSASignature returned '%s', expected nil", err)
```

Comment:
```suggestion
		t.Errorf("Metablock.VerifySignature returned '%s', expected nil", err)
```

Reply:
Fixed in https://github.com/in-toto/in-toto-golang/pull/56/commits/236acb803e2318f44e0c7c32c717e63729c0dd42